server/src/main/java/org/solarbank/server/CurrencyUnitSerializer.java

```diff
@@ -0,0 +1,19 @@
+package org.solarbank.server;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import java.io.IOException;
+import javax.money.CurrencyUnit;
+
+public class CurrencyUnitSerializer extends JsonSerializer<CurrencyUnit> {
```

Comment:
Was unsure about this, added this serialiser so that only the currency code is returned in the service layer response instead of the whole object